src/app/config/projectProperties.json

```diff
@@ -3,17 +3,23 @@
         "angular": {
             "displayname": "Office Add-in project using Angular framework",
             "templates": {
-                "typescript": {
-                    "repository": ""},
                 "javascript": {
+                    "repository": ""},
+                "typescript": {
                     "repository": ""}
-                }        
+                }     
         },
         "excel-functions": {
-            "displayname": "Excel Custom Functions Add-in project (Preview: Requires the Insider channel for Excel)",
+            "displayname": "Excel Custom Functions Add-in project (September 2018 Preview Refresh: Requires the Insider channel for Excel)",
             "templates": {
                 "javascript": {
-                    "repository": "https://github.com/OfficeDev/Excel-Custom-Functions"}
+                    "repository": "https://github.com/OfficeDev/Excel-Custom-Functions",
+                    "branches": [
```

Comment:
shouldn't there just be a single branch name, e.g. ` "branch": "preview-sept-2018-javascript", `